package.json

```diff
@@ -114,9 +115,14 @@
       "not op_mini all"
     ],
     "development": [
+      ">0.2%",
```

Comment:
These are required (can be sorted maybe more nicely. If those are not present, webpack/babel have an issue importing private properties from classes.

Reply:
@nud3l 
Could you please teach me where you got this information?

Reply:
There are some hiccups with polkadot.js and backward compatibility with older versions but I don't think there is any good documentation for it. I just took this from their example substrate UI setup.